client/src/quic_client.rs

```diff
@@ -39,9 +41,14 @@ impl rustls::client::ServerCertVerifier for SkipServerVerification {
         Ok(rustls::client::ServerCertVerified::assertion())
     }
 }
+lazy_static! {
+    static ref RUNTIME: Runtime = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()
```

Comment:
ahh globals!

Reply:
its a thread pool for async green threads, only need one for the entire system